package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "saucelabs-connector",
+  "version": "0.0.1",
+  "dependencies": {
+    "babel-runtime": "^5.8.20",
+    "sauce-tunnel": "^2.2.3",
+    "wd": "^0.4.0"
+  },
+  "devDependencies": {
+    "babel-eslint": "^4.0.5",
+    "es6-promise": "^3.0.2",
+    "es6-promisify": "^3.0.0",
```

Comment:
`es6-promise` and `es6-promisify` are not `devDependencies` but `dependencies` because they are used not only in sources build. To ensure follow the steps:

1) delete the `npm_modules` directory
2) run the `npm install --production` command
3) try to run your project. It'will not work because it's require promise modules